truncator-cronjob/truncate-idx.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -e
+
+# We will see if we required to truncate tables, if we found a large partition (larger than the threshold)
+# proceeed to truncate tables and recreate indices.
+
+# Get one cassandra pod so we can execute nodetool
+CASSANDRA_POD=$(oc get pods -n ${HAWKULAR_NAMESPACE} | grep hawkular-cassandra | head -n 1 |  awk '{print $1}')
```

Comment:
Perhaps, just to be sure, this should check that it picks a Cassandra pod that is in Running state so we don't grab one that is being started/shutting down at that moment. And fail with a reasonable message if there's no suitable pod.

Reply:
Yes, you're right! I'll add a check here.